### PR TITLE
remove controlplane CRUD mode

### PR DESCRIPTION
**Pull request by orozery**
This PR removes the legacy CRUD-based controlplane (i.e. the non-CRD mode).

### CODE REVIEW

File: demos/bookinfo/test.py [hunk 1]
```diff
@@ -14,11 +14,11 @@
 
 ##############################################################################################
 # Name: Bookinfo
-# Info: support bookinfo application with gwctl inside the clusters
+# Info: support bookinfo application inside the clusters
 #       In this we create three kind clusters
-#       1) cluster1- contain gw, gwctl,product and details microservices (bookinfo services)
-#       2) cluster2- contain gw, gwctl, review-v2 and rating microservices (bookinfo services)
-#       3) cluster3- contain gw, gwctl, review-v3 and rating microservices (bookinfo services)
+#       1) cluster1- contain gw, product and details microservices (bookinfo services)
```
Comment (elevran) [inline demos/bookinfo/test.py:19]: nit: unrelated to this change but we should probably be consistent in using lower case or upper case `gw`/`GW` in comments and elsewhere (e.g., here in python vs demos/speedtest/kind/README.md)

**Review by elevran — APPROVED** [review]
Looks good - thanks for getting this large change done!

**Review by praveingk — APPROVED** [review]
Looks good... Just a suggestion. Would be good to preserve the gwctl version of clusterlink, in a separate branch (for a while).

**Review by kfirtoledo — APPROVED** [review]

**Comment by elevran** [comment]
> Just a suggestion. Would be good to preserve the gwctl version of clusterlink, in a separate branch (for a while).

@praveingk release v0.2.1 is tagged and includes the code before the change. A tag can always be converted to a branch.
@orozery please add a tag (e.g., `gwctl-enabled` on the latest `main`, right before merging your PR.

**Pull request merged**